composer.json

```diff
@@ -30,9 +30,10 @@
         "cweagans/composer-patches": "^1.7",
         "drupal/core-composer-scaffold": "^10",
         "drupal/core-recommended": "^10",
-        "drush/drush": "^11",
+        "drush/drush": "^12",
         "morpht/convivial": "dev-main",
-        "oomphinc/composer-installers-extender": "^2"
+        "oomphinc/composer-installers-extender": "^2",
+        "pantheon-systems/quicksilver-pushback": "^2"
```

Comment:
Add morpht/mdt as a dependency to this project. See composer.json of convivial-demo project